### PR TITLE
feature/use absolute import

### PR DESCRIPTION
**Pull request by hsglc**
This PR is related to the #82 . It aims to replace relative import with the absolute imports.

### CODE REVIEW

**Comment by armagandalkiran** [comment]
@hsglc Can you resolve conflicts after that I think we can merge this!

**Comment by hsglc** [comment]
> @hsglc Can you resolve conflicts after that I think we can merge this!

@armagandalkiran it solved. could you check it out

File: gurubu-client/src/app/contexts/GroomingRoomContext.tsx [hunk 1]
```diff
@@ -1,99 +1,106 @@
...
+		</GroomingRoomContext.Provider>
+	);
 }
 
 type GroomingRoomProviderProps = {
-  children: ReactNode;
-  roomId: string;
+	children: ReactNode;
+	roomId: string;
 };
```
Comment (mertcanaltin) [inline gurubu-client/src/app/contexts/GroomingRoomContext.tsx:106]: we can bring back the gaps

File: gurubu-client/src/app/contexts/SocketContext.tsx [hunk 2]
```diff
@@ -1,34 +1,36 @@
...
+	return (
+		<SocketContext.Provider value={socket}>
+			{children}
+		</SocketContext.Provider>
+	);
 }
 
 type SocketProviderProps = {
-  children: ReactNode;
+	children: ReactNode;
```
Comment (mertcanaltin) [inline gurubu-client/src/app/contexts/SocketContext.tsx:35]: here

File: gurubu-client/src/app/create/room/page.tsx [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
-  return (
-    <main className="create-room">
-      <NicknameForm />
-    </main>
-  );
+	return (
+		<main className="create-room">
+			<NicknameForm />
+		</main>
+	);
```
Comment (mertcanaltin) [inline gurubu-client/src/app/create/room/page.tsx:9]: here

File: gurubu-client/tsconfig.json [hunk 4]
```diff
@@ -1,27 +1,33 @@
...
+			"@/components/*": ["components/*"],
+			"@/styles/*": ["styles/*"],
+			"@/contexts/*": ["contexts/*"],
+			"@/shared/*": ["shared/*"],
+			"@/services/*": ["services/*"],
+			"@/room/*": ["room/*"]
+		}
+	},
+	"include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+	"exclude": ["node_modules"]
```
Comment (mertcanaltin) [inline gurubu-client/tsconfig.json:32]: here

**Comment by hsglc** [comment]
> Thank you for developing this topic quickly, can you reinstate the number of spaces of the project, by default we use 2 spaces. here we will make a linter update in the future @hsglc

I changed the indent space to 2 and lint files with it. Could you check it out

**Review by mertcanaltin — APPROVED** [review]
LGTM!

**Review by armagandalkiran — APPROVED** [review]

**Comment by armagandalkiran** [comment]
Merged! Thank you for your contribution! 🚀

**Comment by mertcanaltin** [comment]
🎉

**Pull request merged**